dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/EuTaxonomyData.vue

```diff
@@ -50,7 +60,7 @@ export default {
     async getCompanyInformation() {
       try {
         const metaDataControllerApi = await new ApiClientProvider(this.getKeycloakPromise()).getMetaDataControllerApi();
-        this.metaDataInfo = await metaDataControllerApi.getListOfDataMetaInfo(this.companyID, "EuTaxonomyData");
+        this.metaDataInfo = await metaDataControllerApi.getListOfDataMetaInfo(this.companyID);
```

Comment:
TODO: Calls überprüfen und eventuell neuen Endpoint schaffen

Reply:
Langfristig muss das Framework diese Filterung übernehmen, sodass man hier direkt gefilterte Daten bekommt.
Das hast du völlig recht.
Aber ich denke lieber in einem anderen Ticket (wie besprochen).
Kannst du eins anlegen?

Reply:
Done